osquery/tables/system/darwin/processes.cpp

```diff
@@ -8,7 +8,9 @@
  *  You may select, at your option, one of the above-listed licenses.
  */
 
+#include <kern/kcdata.h>
 #include <libproc.h>
+#include <limits.h>
```

Comment:
Instead of old school `<limits.h>` you could use [<limits>](https://en.cppreference.com/w/cpp/header/limits) from c++11 ;)
It's less ugly just because it is not a macro. More safe because it is templates with good type deduction. And may be a little bit more verbose.

So, instead of `ULLONG_MAX` you can write
```c++
std::numeric_limits<unsigned long long>::max()
std::numeric_limits<std::uint64_t>::max()
```
What do you think?